### ISSUE TITLE
Swift string interpolation causes xcpretty to show provide syntax errors

### ISSUE THREAD

**Issue by kylef** [supermarin/xcpretty#138]
I'm not entirely sure what's going on here, but looks like something is passed to /bin/sh and isn't properly escaped.

#### Output with `| xcpretty -c`

```
sh: -c: line 1: syntax error near unexpected token `)'
sh: -c: line 1: `      XCTFail("Extracted no match template: \(template) with uri: \(uri)")'
sh: -c: line 1: syntax error near unexpected token `)'
sh: -c: line 1: `      XCTFail("Extracted no match template: \(template) with uri: \(uri)")'
sh: -c: line 1: syntax error near unexpected token `)'
sh: -c: line 1: `      XCTFail("Extracted no match template: \(template) with uri: \(uri)")'
sh: -c: line 1: syntax error near unexpected token `)'
sh: -c: line 1: `      XCTFail("Extracted no match template: \(template) with uri: \(uri)")'

URITemplateTests.URITemplateCasesTests
  Level_2_Examples_Case_1_Extraction, failed - Extracted no match template: {+hello} with uri: Hello%20World!
  /Users/kyle/Projects/kylef/URITemplate.swift/URITemplateTests/URITemplateCases.swift:37

  Level_2_Examples_Case_2_Extraction, failed - Extracted no match template: {+path}/here with uri: /foo/bar/here
  /Users/kyle/Projects/kylef/URITemplate.swift/URITemplateTests/URITemplateCases.swift:37

  Level_2_Examples_Case_3_Extraction, failed - Extracted no match template: here?ref={+path} with uri: here?ref=/foo/bar
  /Users/kyle/Projects/kylef/URITemplate.swift/URITemplateTests/URITemplateCases.swift:37

  Level_1_Examples_Case_1_Extraction, failed - Extracted no match template: {hello} with uri: Hello%20World%21
  /Users/kyle/Projects/kylef/URITemplate.swift/URITemplateTests/URITemplateCases.swift:37

	 Executed 242 tests, with 4 failures (0 unexpected) in 0.213 (0.276) seconds
```

**Comment by supermarin** [comment]
@kylef let's check this out in person this week, just remind me please :angel:

**Comment by kattrali** [comment]
Fixed by #141
